portal-web/docroot/html/taglib/ui/page_iterator/start.jsp

```diff
@@ -410,6 +414,77 @@ NumberFormat numberFormat = NumberFormat.getNumberInstance(locale);
 	}
 </script>
 
+<script data-senna-track="temporary" type="text/javascript">
+	var dropdown = document.getElementById("<%= ariaPagination %>");
+
+	var button = dropdown.querySelector('.dropdown-toggle');
+	var list = dropdown.querySelector('.dropdown-menu');
+
+	var options = list.querySelectorAll('.dropdown-item');
+	var selectedItemValue = button.dataset.attribute;
+
+	function onButtonKeyDown(event) {
+		if (event.key === 'ArrowDown' || event.key === 'ArrowUp' || event.key === 'Enter' || event.key === 'Space') {
+			event.preventDefault();
+
+			list.classList.add('show');
+			button.setAttribute('aria-expanded', 'true');
+
+			var selectedOption = list.querySelector('.active');
+
+			if (selectedOption) {
+				selectedOption.focus();
+			}
+		}
+	}
+
+	button.addEventListener('keydown',onButtonKeyDown );
+
+	function onLeaveDropdown() {
+		list.classList.remove('show');
+		button.setAttribute('aria-expanded', 'false');
+	}
+
+	function handleKeyEvents(event) {
+		var currentIndex = Array.from(options).indexOf(document.activeElement);
+
+		if (event.key === 'ArrowDown') {
+			event.preventDefault();
+
+			if (currentIndex < options.length - 1) {
+				options[currentIndex + 1].focus();
+			}
+		} else if (event.key === 'ArrowUp') {
+			event.preventDefault();
+
+			if (currentIndex > 0) {
+				options[currentIndex - 1].focus();
+			}
+		} else if (event.key === 'Escape') {
+			onLeaveDropdown()
+			button.focus()
```

Comment:
Same thing... these should either be sorted or have a empty line between them to indicate that the order matters.

Reply:
Fixed.